### PR TITLE
Add Idea class and update main and index

### PR DESCRIPTION
**Pull request by tanya2108**
What (if any) features are you implementing?
-Adding Idea class
Were there any issues that arose?
-forgot to pull main
Is there anything that you need from your teammate?
-pseudocode, plan for iteration2

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1 @@
+var ideas = [];
```
Comment (minsungpark89) [inline main.js:1]: Maybe need to change name of variable to be more unique.

File: Idea.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+class Idea {
```
Comment (JRicardoT) [inline Idea.js:1]: Class has all properties needed for the following iteration.

Comment (minsungpark89) [inline Idea.js:1]: Nice job on the class!

**Review by JRicardoT — COMMENTED** [review]
Everything looks good. Seems ready to merge.

**Review by minsungpark89 — COMMENTED** [review]
Good job Tanya, left some comments we can review later!

**Pull request merged**